### PR TITLE
Adding multiple emails and phones for person #485

### PR DESCRIPTION
**Pull request by SGospodinov**

### CODE REVIEW

**Comment by kgf** [comment]
Thanks for your help, @SGospodinov !
Was upgrading the version of Rails a requirement ? I would prefer that to be in a separate PR, to make it more apparent.

**Comment by SGospodinov** [comment]
@kgf, it wasn't. I did it by mistake. Do you want me to change it and rebase the commit?

File: app/views/phones/_phone_fields.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<div class="nested-fields">
+  <fieldset>
+    <%= phone.input :id, :as => :hidden %>
+    <%= phone.input :category, :as => :select, :collection => Channel::CATEGORIES %>
+    <%= phone.input :content, :placeholder => '9785551212' %>
```
Comment (kgf) [inline app/views/phones/_phone_fields.html.erb]: Could you please change this to "number" ? See issue #482 
Also, the placeholder text should be +19995551212 ? See issue #483 
Thanks !

Comment (kgf) [reply app/views/phones/_phone_fields.html.erb]: If you could revert the Rails upgrade, and the two other small changes, that would be great. Thanks for your help!

**Comment by kgf** [comment]
Thanks for those changes, the phone looks good now. But we still have the Rails upgrade.

**Comment by kgf** [comment]
Whoa, that broke a few tests !!

**Comment by SGospodinov** [comment]
@kgf, now everything works fine. The gems version change was causing the problem. The rails update is opened as a separate issue (#450), so we could discuss it there.

**Pull request merged**